assets/src/edit-story/components/library/panes/media/common/paginatedMediaGallery.js

```diff
@@ -141,6 +165,12 @@ function PaginatedMediaGallery({
     };
   }, [handleScrollOrResize]);
 
+  const openLink = useCallback(() => {
+    if (providerType === ProviderType.UNSPLASH) {
+      window.open('https://unsplash.com');
```

Comment:
Shall this open in a new tab / window?

Reply:
Why window.open and not just a hyperlink? Would be better for accessibility for sure.

Reply:
Good point, updated.